runtime/oti/util_api.h

```diff
@@ -2197,6 +2197,11 @@ getJVMFeature(J9JavaVM *vm);
 uint64_t
 getOpenJ9Sha();
 
+/**
+ * @return pointer to the last '$' sign of the class name if it is a lambda class, NULL otherwise
+ */
+char* getEndOfLambdaClassName(const char *className, UDATA classNameLength);
```

Comment:
Probably a better name for this function is getLastDollarSignOfLambdaClassName().